business-central-parent/business-central-home-page-community/src/main/resources/org/kie/bc/client/resources/i18n/Constants_fr.properties

```diff
@@ -15,7 +15,7 @@
 #
 
 Heading=Bienvenue dans Business Central
-SubHeading=Business Central propose un ensemble d’outils flexibles qui s’adaptent à votre façon de travailler. Sélectionnez un outil ci-dessous pour commencer.
+SubHeading=Business Central propose un ensemble d'outils flexibles qui s'adaptent à votre façon de travailler. Sélectionnez un outil ci-dessous pour commencer.
```

Comment:
Interestingly @mbiarnes the translators have changed from right single quotation (`u2019`) to an apostrophe!

So... the `LoginConstants` may well change back in the future.

Reply:
maybe it is a change by mistake ?